python/paddle/fluid/layers/nn.py

```diff
@@ -3759,3 +3760,44 @@ def label_smooth(label,
         outputs={"Out": smooth_label},
         attrs={"epsilon": float(epsilon)})
     return smooth_label
+
+
+def roi_pool(input, rois, pooled_height=1, pooled_width=1, spatial_scale=1.0):
+    """
+    RoI pooling, its purpose is to perform max pooling on inputs of nonuniform sizes to obtain fixed-size feature maps (e.g. 7*7).
+    The operator has three steps: 
+        1.Dividing each region proposal into equal-sized sections with the pooled_width and pooled_height 
+        2.Finding the largest value in each section 
+        3.Copying these max values to the output buffer
+
+    Args:
+        input (Variable): The input for roi pooling.
+        rois (Variable): ROIs (Regions of Interest) to pool over. It should be a 2-D 1-level lod tensor of shape (num_rois, 4). It given as [[x1, y1, x2, y2], ...], where (x1, y1) is the top left coordinates, and (x2, y2) is the bottom right coordinates.
+        pooled_height (integer): The pooled output height. Default: 1
+        pooled_width (integer): The pooled output width. Default: 1
+        spatial_scale (float): Multiplicative spatial scale factor. To translate ROI coords from their input scale to the scale used when pooling. Default: 1.0
+
+    Returns:
+        pool_out (Variable): The output of ROIPoolOp is a 4-D tensor with shape (num_rois, channels, pooled_h, pooled_w).
```

Comment:
The output is a 4-D tensor of the shape (num_rois, channels, pooled_h, pooled_w).

Reply:
ok，this describe is more clear。